requirements.txt

```diff
@@ -4,4 +4,5 @@ jupyter>=1.0.0,<2
 jupyter_contrib_nbextensions>=0.5.1,<1
 jupyterlab>=3.1.5,<4
 numpy>=1.22,<2
-watchdog>=2.1.6,<3
\ No newline at end of file
+watchdog>=2.1.6,<3
+traitlets==5.2.2.post1
```

Comment:
Where is this requirement from?

Reply:
Earlier today traitlets updated causing various parts of iPython to break. [This was a quick fix that they made](https://github.com/ipython/traitlets/issues/740).